kafka-impl/src/main/java/io/streamnative/pulsar/handlers/kop/KopEventManager.java

```diff
@@ -90,46 +105,61 @@ public void close() {
     }
 
 
-    public void put(KopEvent event) {
+    public void put(KopEventWrapper eventWrapper) {
         putLock.lock();
         try {
-            queue.put(event);
+            queue.put(eventWrapper);
+            KopEventManagerStats.KOP_EVENT_QUEUE_SIZE_INSTANCE.incrementAndGet();
         } catch (InterruptedException e) {
             Thread.currentThread().interrupt();
-            log.error("Error put event {} to coordinator event queue", event, e);
+            log.error("Error put event {} to coordinator event queue",
+                    eventWrapper.toString(), e);
         } finally {
             putLock.unlock();
         }
     }
 
-    public void clearAndPut(KopEvent event) {
+    public void clearAndPut(KopEventWrapper eventWrapper) {
         putLock.lock();
         try {
             queue.clear();
-            put(event);
+            KopEventManagerStats.KOP_EVENT_QUEUE_SIZE_INSTANCE.set(0);
+            put(eventWrapper);
+            KopEventManagerStats.KOP_EVENT_QUEUE_SIZE_INSTANCE.incrementAndGet();
```

Comment:
Why do we increment twice? See `put(KopEventWrapper eventWrapper)`

Reply:
Sorry, this is my mistake, thanks